packages/worker/src/workflow/bundler.ts

```diff
@@ -221,11 +219,11 @@ export { api };
       },
       entry: [entry],
       mode: 'development',
-      devtool: 'source-map',
+      devtool: 'inline-source-map',
       output: {
+        pathinfo: true,
```

Comment:
Note to myself: not sure if this is rquired

Reply:
Want to check?